docs/installation.md

```diff
@@ -5,86 +5,66 @@ tableOfContents: true
 # Installation
 
 ## Introduction
+
 Tiptap is framework-agnostic and even works with Vanilla JavaScript (if that’s your thing). The following integration guides help you integrating Tiptap in your JavaScript project.
 
 ## Integration guides
+
 <!-- * [CDN](/installation/cdn) -->
```

Comment:
I think the idea was to reactivate a CDN solution in future. Maybe we keep this?

Reply:
good point